### PR TITLE
Passwordless user type clarification

### PR DESCRIPTION
**Pull request by stevenGravy**
Provide clarification on user types supported

Mark tsh download package provided by Teleport as signed.  Matches language on downloads page and instructions from Passwordless page.

### CODE REVIEW

**Review by codingllama — APPROVED** [review]
Thanks, Steven!

File: docs/pages/access-controls/guides/passwordless.mdx [hunk 1]
```diff
@@ -9,7 +9,8 @@ videoBanner: GA37qqB6Lmk
 </Admonition>
 
 Passwordless takes advantage of WebAuthn to provide passwordless and
-usernameless authentication for Teleport.
+usernameless authentication for Teleport.  Teleport [local users](../../setup/admin/users.mdx) are the only initial
```
Comment (zmb3) [inline docs/pages/access-controls/guides/passwordless.mdx:12]: Something about the verbiage here doesn't sound quite right to me.

How about `Passwordless authentication is support for local users only` instead?

Comment (ptgott) [reply docs/pages/access-controls/guides/passwordless.mdx:12]: I'm not sure the introductory paragraph of this guide is the best place to put this caveat. In this PR, half of the intro is a statement about what passwordless is and half is a specific warning about usage. In an introduction, I would expect a general explanation of what the feature is why you would use it. 

Maybe add a note like this to the Prerequisites instead:

```markdown
- At least one local Teleport user. Passwordless currently
only supports authentication for local users, and does 
not support single sign-on connectors.
```

File: docs/pages/installation.mdx [hunk 2]
```diff
@@ -135,7 +135,7 @@ chart.
   |Link|Binaries|
   |-|-|
   |[`teleport-(=teleport.version=).pkg`](https://get.gravitational.com/teleport-(=teleport.version=).pkg)|`teleport`<br/>`tctl`<br/>`tsh`<br/>`tbot`|
-  |[`tsh-(=teleport.version=).pkg`](https://get.gravitational.com/tsh-(=teleport.version=).pkg)|`tsh`|
+  |[`tsh-(=teleport.version=).pkg`](https://get.gravitational.com/tsh-(=teleport.version=).pkg) (signed)|`tsh`|
```
Comment (ptgott) [inline docs/pages/installation.mdx:138]: Are both the `teleport` and `tsh` installers signed? If so, I think we should put that information in the sentence before the table. If not, this is okay as-is.

**Comment by ptgott** [comment]
@stevenGravy double-checking if you still want to merge this one

**Pull request closed without merging**